instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import CommentInput from "./CommentInput";
+import Comment from "./Comment";
+import moment from "moment";
+import styled from 'styled-components';
+
+const CommentsSection = styled.section`
+    display: flex;
+    flex-direction: column;
+`;
+
+const TimeStamp = styled.span`
+    color: silver;
+    font-size: .8rem;
+    padding-left: 15px;
+`;
+
+const BorderTop = styled.hr`
+    color: rgb(199, 198, 198);
+    width: 96%;
+    margin: 15px auto; 
+`;
+
+class CommentSection extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: props.comments,
```

Comment:
I'm hoping you understand why we wouldn't want to keep the comments array in state here. Ideally this would be help on `PostPage` and updated there. By only updating the data here we get out of sync with the rest of our app.